build.sh

```diff
@@ -403,6 +423,22 @@ node --version
   make install SUBDIRS='libtiff' noinst_PROGRAMS= dist_doc_DATA=
 )
 
+[ -f "$TARGET/lib/libresvg.a" ] || [ -n "$DISABLE_SVG" ] || (
+  stage "Compiling resvg"
+  mkdir -p $DEPS/resvg
+  curl -Ls https://github.com/RazrFalcon/resvg/releases/download/v$VERSION_RESVG/resvg-$VERSION_RESVG.tar.xz | tar xJC $DEPS/resvg --strip-components=1
+  cd $DEPS/resvg/c-api
+  # Vendor dir doesn't work with -Zbuild-std right now due to https://github.com/rust-lang/wg-cargo-std-aware/issues/23
+  # Just delete the config so that all deps are downloaded off the internet.
+  rm ../.cargo/config
+  # We don't want to build the shared library
+  sed -i 's/crate-type = .*/crate-type = ["staticlib"]/' Cargo.toml
+  # Note: --release dosn't work right now due to https://github.com/rust-lang/rust/issues/91628
+  cargo build --release --target wasm32-unknown-emscripten --locked -Zbuild-std=panic_abort,std --no-default-features --features filter,raster-images
```

Comment:
Perhaps we should also build with `-Zbuild-std-features=panic_immediate_abort`?
```suggestion
  cargo build --release --target wasm32-unknown-emscripten --locked -Zbuild-std=panic_abort,std -Zbuild-std-features=panic_immediate_abort --no-default-features --features filter,raster-images
```
(untested)

Reply:
It's possible, but the point of that feature is that it swallows the error messages when something does panic. Given that panics occur only on critical bugs, personally I value knowing what happened more than the savings it provides, but I can add it if you think it's worth it.

Reply:
Ah, I see. I don't think we actually need that optimization flag, given that `vips-resvg.wasm` is only 1.4 MB (making it the smallest dynamic module in size so far).
```console
$ ls -lh lib/*.wasm | awk '{print $5,$9}'
5.0M lib/vips-heif.wasm
2.0M lib/vips-jxl.wasm
1.4M lib/vips-resvg.wasm
5.5M lib/vips.wasm
```